### PR TITLE
create ManageAssignedAssessments and ViewAssignedAssessments scopes

### PR DESCRIPTION
**Pull request by abrantesarthur**
## Related Issues

- closes https://transcend.height.app/T-32947

### CODE REVIEW

**Review by giacaglia — APPROVED** [review]

File: src/scopes.ts [hunk 1]
```diff
@@ -668,6 +670,20 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
     type: ScopeType.Modify,
     products: [TranscendProduct.Assessments, TranscendProduct.DataMapping],
   },
+  [ScopeName.ViewAssignedAssessments]: {
+    title: 'View Assessments',
```
Comment (iamtheluckyest) [inline src/scopes.ts:674]:
```suggestion
    title: 'View Assigned Assessments',
```

File: src/scopes.ts [hunk 2]
```diff
@@ -668,6 +670,20 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
...
+    dependencies: [],
+    description: 'View the asssigned assessments forms.',
+    type: ScopeType.View,
+    products: [TranscendProduct.Assessments, TranscendProduct.DataMapping],
+  },
+  [ScopeName.ManageAssignedAssessments]: {
+    title: 'Manage Assessments',
```
Comment (iamtheluckyest) [inline src/scopes.ts:681]:
```suggestion
    title: 'Manage Assigned Assessments',
```

File: src/scopes.ts [hunk 3]
```diff
@@ -668,6 +670,20 @@ const SCOPES_WITHOUT_VIEW_ONLY: {
     type: ScopeType.Modify,
     products: [TranscendProduct.Assessments, TranscendProduct.DataMapping],
   },
+  [ScopeName.ViewAssignedAssessments]: {
+    title: 'View Assessments',
+    dependencies: [],
+    description: 'View the asssigned assessments forms.',
```
Comment (iamtheluckyest) [inline src/scopes.ts:676]:
```suggestion
    description: 'View the assigned assessments forms.',
```

**Review by iamtheluckyest — COMMENTED** [review]
Ah, you were too fast again! 😂

File: src/scopes.ts (hunk 1 above)
Comment (abrantesarthur) [reply src/scopes.ts:674]: damn, sorry haha opening a new one. Thanks for catchin!

**Pull request merged**